packages/app/src/cli/services/dev/select-app.test.ts

```diff
@@ -66,7 +66,7 @@ describe('createApp', () => {
     vi.mocked(appNamePrompt).mockResolvedValue('app-name')
     vi.mocked(partnersRequest).mockResolvedValueOnce({appCreate: {app: APP1, userErrors: []}})
     const variables = {
-      org: 2,
+      org: 1,
```

Comment:
just for my own learnings, why are we changing the org here?

Reply:
The test has been working by chance 😓 . 
- [APP1](https://github.com/Shopify/cli/blob/main/packages/app/src/cli/services/dev/select-app.test.ts#L43) organization is set to `1`
- Variables uses `org: 2`
- This was a piece of code previous to my change where the `APP1` is modified using the `Variables` values (mutations, oh mutations 😅 )
```ts
  const createdApp: OrganizationApp = result.appCreate.app
  createdApp.organizationId = org.id
  createdApp.newApp = true
  return createdApp 
```
- The new code is not using mutation but a new copy of `APP1` with the modified values (the org is not mutable)
```
  const betas = filterDisabledBetas(result.appCreate.app.disabledBetas)
  return {...result.appCreate.app, organizationId: org.id, newApp: true, betas}
```